crates/forge/tests/e2e/color.rs

```diff
@@ -0,0 +1,32 @@
+use snapbox::cmd::{cargo_bin, Command as SnapboxCommand, OutputAssert};
+
+use crate::e2e::common::runner::setup_package;
+
+fn runner_color(value: &str) -> SnapboxCommand {
+    SnapboxCommand::new(cargo_bin!("snforge"))
+        .arg("test")
+        .arg("--color")
+        .arg(value)
+}
+
+fn has_escape(output: &OutputAssert) -> bool {
+    String::from_utf8(output.get_output().stdout.clone())
+        .unwrap()
+        .contains("\x1b[")
+}
+
+#[test]
+fn color_always() {
+    let temp = setup_package("simple_package");
+    let snapbox = runner_color("always");
+    let output = snapbox.current_dir(&temp).assert().code(1);
+    assert!(has_escape(&output), "Stdout doesn't contains escape code");
```

Comment:
Better error message would be `"output expected to be colored but it is not"`. Analogous above

Reply:
Done